exercises/concept/amusement-park/.docs/instructions.md

```diff
@@ -0,0 +1,141 @@
+- The identifier of the ticket that the visitor bought.
+
+The function should return an object that holds this information.
+The keys in the object should be called `name`, `age` and `ticketId`.
+
+```javascript
+createVisitor('Verena Nardi', 45, 'H32AZ123');
+// => { name: 'Verena Nardi', age: 45, ticketId: 'H32AZ123' }
+```
+
+## 2. Revoke the ticket
+
+When people leave the amusement park, their ticket gets revoked.
+If they come back, they need to buy a new one.
+To save regular visitors some time, they only need to register once and the visitor information will be kept for subsequent visits.
+
+That means when a visitor leaves the park, only their ticket should be invalidated but the rest of the visitor object should stay the same.
+Implement the function `revokeTicket` that accepts a visitor object, sets the ticket identifier to `null` and returns the object afterwards.
+
+```javascript
+const visitor = {
+  name: 'Verena Nardi',
+  age: 45,
+  ticketId: 'H32AZ123',
+};
+
+revokeTicket(visitor);
+// => { name: 'Verena Nardi', age: 45, ticketId: null }
+```
+
+## 3. Determine the ticket status
+
+To prevent forgery, the ticket identifiers are unique.
+Once a ticket was printed, the identifier is added as key in an object to track it in the system.
+
+Before the ticket is sold to a visitor, it is stored with the value `null` in the ticket tracking object.
+Once it was sold to a visitor, the visitor's name will be assigned as a value.
+When employees have doubts about the validity of a ticket, they need to check the status of the ticket in the system.
+
+Implement a function `ticketStatus` that accepts the tracking object and a ticket identifier as arguments.
+It should return one of the following results.
+
+- `unknown ticket id` if the identifier was not found in the tracking object
+- `not sold` in case the ticket was printed but not bought by visitor yet
+- `sold to ` followed by the name of the visitor if the ticket was sold
+
+```javascript
+const tickets = {
+  '0H2AZ123': null,
+  '23LA9T41': 'Verena Nardi',
+};
+
+ticketStatus(tickets, 'RE90VAW7');
+// => 'unknown ticket id'
+
+ticketStatus(tickets, '0H2AZ123');
+// => 'not sold'
+
+ticketStatus(tickets, '23LA9T41');
+// => 'sold to Verena Nardi'
+```
+
+## 4. Improve the ticket status response
+
+After a while, you get feedback from the employees that they want the ticket status to be easier to understand at the first glance.
+They only want to see either the name of the visitor or that the ticket is invalid.
+
+Write a function `simpleTicketStatus` that accepts the same arguments as `ticketStatus` in task 3.
+This function only returns one of two different results.
+
+- the name of the visitor if the ticket was sold
+- `invalid ticket !!!` if the ticket was not sold yet or the identifier was not found in the tracking object
+
+Note that `invalid ticket !!!` should not be returned for any "strange" name values that might occur but really only in the two cases described above.
```

Comment:
Is `any "strange" name values that might occur` a reference to the tests? This entire sentence might be more confusing than clarifying.

Reply:
You are right. I removed the sentence. The tests will speak for themselves.